packages/forms-web-app/src/dynamic-forms/dynamic-components/boolean-text/question.js

```diff
@@ -34,6 +35,8 @@ class BooleanTextQuestion extends RadioQuestion {
 			options,
 			validators
 		});
+
+		this.html = html;
```

Comment:
arguably we want to inherit this from parent class(es)? the html field hasn't been universally implemented yet